packages/amplify-e2e-core/src/categories/api.ts

```diff
@@ -477,3 +477,34 @@ export function addApiWithCognitoUserPoolAuthTypeWhenAuthExists(projectDir: stri
       });
   });
 }
+
+export function addRestContainerApi(projectDir: string) {
+  return new Promise((resolve, reject) => {
+    spawn(getCLIPath(), ['add', 'api'], { cwd: projectDir, stripColors: true })
+      .wait('Please select from one of the below mentioned services:')
+      .sendKeyDown()
+      .sendCarriageReturn()
+      .wait('Which service would you like to use')
+      .sendKeyDown()
```

Comment:
Can you use `singleSelect` like instead of suing `sendKeyDown`
https://github.com/aws-amplify/amplify-cli/blob/977c84be0e16aa7727a9acdf41bf9c51fa430bed/packages%2Famplify-e2e-core%2Fsrc%2Fcategories%2Flambda-function.ts#L61-L69